log-manager/src/main/java/io/airlift/log/Logging.java

```diff
@@ -194,50 +114,20 @@ public void setLevels(File file)
         processLevels(properties);
     }
 
+    @SuppressWarnings("MethodMayBeStatic")
```

Comment:
This method has been an instance method for a while.  I can change it to a static method, but it will break any current users.  Is this a change we want to make?